.travis.yml

```diff
@@ -1,13 +1,45 @@
 language: java
 
+python:
+  - "2.7"
+
 sudo: required
 
 services:
   - docker
 
+notifications:
+#  email:
+#    recipients:
+#      - username@email.dom
+#    on_success: never # default: change
+#    on_failure: always # default: always
+
+env:
+  global:
+#    Please see https://docs.travis-ci.com/user/encryption-keys/ for information
+#    about adding encryption keys which can be used to store auth tokens used in running apps against
+#    the KBase infrastructure
+#    secure: "blahblahblah"
+
+before_install:
+  - env
+
 install:
-  - sudo pip install nose2
+  - pushd ..
+  - git clone https://github.com/kbase/jars
+  - popd
 
 script:
-  - make
+  - make bin
```

Comment:
I believe this will only test the prebuilt jar file. I'm pretty sure we should be testing the latest code, not just whatever was previously built.